tests/e2e/trustyai_test.go

```diff
@@ -75,3 +80,60 @@ func (c *TrustyAITestCtx) disableKserve(t *testing.T) {
 		BeEmpty(),
 	)
 }
+
+func (c *TrustyAITestCtx) validateCRDReinstated(t *testing.T) {
+	// validate multikuueclusters, multikueueconfigs has v1beta1 version
+
+	crds := []string{
+		"inferenceservices.serving.kserve.io", // SR is not needed any more in 2.18.0
```

Comment:
is this "SR is not needed any more in 2.18.0" tracked anywhere ?

Reply:
in TrustyAI's role binding I still see:

```yaml
- apiGroups:
  - serving.kserve.io
  resources:
  - inferenceservices/finalizers
  verbs:
  - delete
  - get
  - list
  - patch
  - update
  - watch
- apiGroups:
  - serving.kserve.io
  resources:
  - servingruntimes
  verbs:
  - create
  - delete
  - get
  - list
  - patch
  - update
  - watch
```

Reply:
they have not merged https://github.com/trustyai-explainability/trustyai-service-operator/pull/400 , 
so our PR need to wait till that gets in ODH first

Reply:
put "WIP" on the PR now